### DISCUSSION TITLE
(Places/Map), filter for photos in region

### DISCUSSION THREAD

**Discussion by lennartbrandin** [photoprism/photoprism#2634] [Ideas]
I would like to be able to filter for pictures from a specific region by clicking the picture count bubble from (Places/Map)

**Current behavior**

You zoom further in when clicking on a bubble with the count of pictures in the region.

**New behavior**

You click a bubble with the count of pictures and open all covered by the counter. (Perhaps filter with latitude/longitude)
![image](https://user-images.githubusercontent.com/92579517/185932544-232d3122-8844-4a87-bc49-fb1e0d6ecb55.png)

![image](https://user-images.githubusercontent.com/92579517/185931840-66961821-52b0-4e4a-be68-ea51916b9e1d.png)

**Comment by graciousgrey** [comment]
We already have a related issue https://github.com/photoprism/photoprism/issues/1187 :)
